app/models/project.rb

```diff
@@ -7,11 +7,13 @@ class Project < ApplicationRecord
   validates :trust_ukprn, presence: true, numericality: {only_integer: true}
   validates :target_completion_date, presence: true
   validates :team_leader, presence: true
+  validates :regional_delivery_officer_id, presence: true, allow_blank: false
   validate :establishment_exists, :trust_exists, on: :create
   validate :target_completion_date, :first_day_of_month
 
   belongs_to :delivery_officer, class_name: "User", optional: true
   belongs_to :team_leader, class_name: "User", optional: false
+  belongs_to :regional_delivery_officer, class_name: "User", optional: true
```

Comment:
This is really annoying, if its `optional: false`, it adds a validation error on the `regional_delivery_officer` attribute.

Having looked how people generally deal with this, they tend to just set `optional: true` and set the validation on the foreign key instead like I've done here

Reply:
I think there are two options for us here.

1. treat `regional_delivery_officer` like `delivery_officer` i.e allow it to not be set `optional: true` and no db constraint
2. assume `regional_delievery_officer` must always be set like `team_lead`

Our thinking has moved on slightly since we wrote this card, so I think it would be good to align this work now - shall we have a chat?

Reply:
We've discussed this and as we know more about what this will look like moving forward, we can put this in and come back and refactor.